src/main/java/org/datadog/jenkins/plugins/datadog/events/ComputerOfflineEventImpl.java

```diff
@@ -34,6 +34,8 @@ of this software and associated documentation files (the "Software"), to deal
 
 public class ComputerOfflineEventImpl extends AbstractDatadogSimpleEvent {
 
+    private boolean isTemporarily;
```

Comment:
**question:** ‏ should this be moved to the Parent abstract class?

Reply:
moved!